python/docs/logging.rst

```diff
@@ -0,0 +1,31 @@
+.. currentmodule:: dlisio
+
+Logging
+=======
+While working with dlisio, users should expect to receive log information from
+the package.
+Some of this information users have certain control over (:class:`common.ErrorHandler`),
+while some is not customizable.
+
+In both cases, when occurred event is recoverable,
+`default python logging module <https://docs.python.org/3/library/logging.html#module-logging>`_
+is used to report it.
+
+Users have to
+`configure logging <https://docs.python.org/3/howto/logging.html#configuring-logging/>`_
+themselves.
+If user doesn't provide any,
+`default configuration <https://docs.python.org/3/howto/logging.html#what-happens-if-no-configuration-is-provided/>`_
+applies.
```

Comment:
This confuses me a bit. It's hard to tell which 'control' you are referring too. As this is the logger section, I would expect that the docs here refers to controlling the logging module. But it's actually talking about ErrorHandler. 

Would probably drop all talk of ErrorHandler, to limit the confusion, and simply talk about how dlisio used Pythons logger module, and how it should be configured. I think this should cover it:

> dlisio is using Python's standard library logging module to emit several logs. dlisio only supply the loggers, while [configuring](https://docs.python.org/3/howto/logging.html#configuring-logging) them is left as an exercise to the user. If not, the logging modules [default configuration](link) applies.